### ISSUE TITLE
Got it running, a few notes and questions/issues

### ISSUE THREAD

**Issue by MrJamesEllis** [amzn/alexa-avs-raspberry-pi#41]
This is great, got it up and running after some hiccups...

Questions:
- Any settings for getting it answering in metric? Getting told the temperature in F makes no sense if you aren't from the US. Would be nice to have some locale and location settings.
- If I ask "What's the temperature in Sydney?" It'll generally cut me off at the S and give me the weather in Seattle (in F). Are there any settings for getting it to be a bit more patient?

A few notes that I made during setup to help others:

- Not sure if Oracle Java needs to be installed twice at 2.6 and 9 ? This seems to install the JDK twice?
- I installed Node 5.x, not the version specified at step 2.5 (see https://nodejs.org/en/download/package-manager/)
- I kept on getting a general error after hitting Okay at step 10.3. Going back through the steps I changed:
 - the privacy policy URL to example.com (I'd put local.dev in)
 - I regenerated the SSL cert using a passphrase which I didn't specify initially
 - I had the productId and products settings in config.json and configs.js set incorrectly.

Fixing one or more of those items, I think the last was the most important, got me past the general error after step 10.3 and to the device tokens ready at 10.4

Hope this helps others who may be stuck, it's a good little long weekend project.

**Comment by kNeerajPro** [comment]
Great hack man!!
How are you getting the output form the Raspberry pi? Through earphones/headphones or speakers? 
Is voice loud enough for speakers?

**Comment by stphnlee** [comment]
There is an option to change to metric measurements through the settings menu of the Amazon Alexa companion app.

**Comment by gregwa1953** [comment]
Does anyone know where the Amazon Alexa Companion App would be found?

**Comment by gregwa1953** [comment]
Ahhhh...found it.  It's in the Google play store if you have an Android smart phone.  I'm guessing it's also in the apple store as well.

**Comment by ajotwani** [comment]
@gregwa1953 You can also visit alexa.amazon.com on your browser.

**Comment by gregwa1953** [comment]
Thank you.

**Comment by MrJamesEllis** [comment]
Thanks, but I can't access the Alexa app even over a US VPN, availability must be determined by a billing address :(

Found the setting on alexa.amazon.com, it's per device under Settings - <select device>


@kNeerajPro I had to force output through the audio jack using raspi-config - it's under "Advanced". If you don't do this and have the HDMI plugged in it'll output audio over the HDMI (I think).

**Comment by MrJamesEllis** [comment]
Actually you can't set an address or turn the metric measurements setting on via alexa.amazon.com, with the Alexa Device set up and "Online".

Metric Measurements: "Unable to save your setting. See Help for troubleshooting."

Device Location -  enter address or choose from list provided - error is "Could not set your address. Please ensure you have correctly entered a United States address, and included street name, city, and state."

I guess we're stuck 2 miles north of Seattle in Fahrenheit.

**Comment by gregwa1953** [comment]
Have you tried the smartphone app?

**Comment by gregwa1953** [comment]
@JamesEllis,
Try with Alexa offline.  If not, I'm sure that they will support other
locales than US soon.

**Comment by Muurtegel** [comment]
I don't think you can download the smartphone app outside of the US. I've tried the Apple App Story and couldn't find anything. I'm in Europe.

**Comment by gregwa1953** [comment]
hmmmmmmmmm

There is a link for "help and feedback" at http://echo.amazon.com/#help.
Might be worth a shot?

**Comment by Muurtegel** [comment]
> The item you've requested is not currently available in the Dutch Store, but it is available in the U.S. Store. Click Change Store to view this item.

Changing to the US Store didn't work with my Apple Account unfortunately.

**Comment by MrJamesEllis** [comment]
At least for Android you need a US Billing or Shipping address in your Google settings.

On alexa.amazon.com, I found that I could finally get it on to metric by entering a post code in Guam (my timezone)  and saving that. Picking an address in Guam from the autosuggest  resulted in a "Could not set your address...." error message.
Once a post code was set, I could set the "Metric Measurements" to On.... I found this out by having a look at the XHR request in Chrome and seeing the message "Postal code cannot be blank or null' before the post code had been saved.

So at least we're now in standard measurements but I'll be getting the local weather and traffic for Guam :smile: 

Never understood this whole US-centric rollout thing, it's like being told to look something interesting then getting the bird.

**Comment by ajotwani** [comment]
We're closing this Issue since the original poster (or another commenter) hasn't yet responded to the question or request made to them 14 days ago. We therefore assume that the user has lost interest or resolved the problem on their own.

Don't worry though; if this is an error, let us know with a comment and we'll be happy to reopen the Issue.

Thanks!

**Comment by ajotwani** [comment]
Thanks @MrJamesEllis for these notes. Will be sure to include them in the next update. Thanks!

**Comment by yozoyguapo** [comment]
well , 3/ march /  2017 and the issue seems still happening, 
1) i cant use the app in my iphone (i am in Ireland)
2) using the website does not allow the address to be input (eventhough the address is found)
